src/Tickets/Commerce/Gateways/Square/Notices/Webhook_Notice.php

```diff
@@ -0,0 +1,207 @@
+		if (
+			empty( $status ) ||
+			(
+				( isset( $status['is_healthy'] ) && $status['is_healthy'] ) &&
+				( isset( $config['is_current'] ) && $config['is_current'] )
+			)
+		) {
+			return false;
+		}
+
+		// Only show on admin pages
+		if ( ! is_admin() ) {
+			return false;
+		}
+
+		// Don't show on tickets admin pages where we already have inline notices
+		$screen = get_current_screen();
+		if ( $screen && $this->is_tickets_admin_page( $screen->id ) ) {
+			return false;
+		}
+
+		return true;
+	}
+
+	/**
+	 * Render the webhook notice
+	 *
+	 * @since TBD
+	 *
+	 * @return string
+	 */
+	public function render_notice() {
+		$health_status = tribe_get_option( Webhooks::$option_webhook_last_check, [] );
+		$config_status = tribe_get_option( Webhooks::$option_webhook_configuration, [] );
+		$webhook_id = tribe_get_option( Webhooks::$option_webhook_id );
+
+		$issues = [];
+
+		// Check if webhook is missing
+		if ( empty( $webhook_id ) ) {
+			$issues[] = esc_html__( 'Webhook not registered', 'event-tickets' );
+		}
+
+		// Check health status
+		if ( ! empty( $health_status ) && isset( $health_status['is_healthy'] ) && ! $health_status['is_healthy'] ) {
+			$issues[] = esc_html__( 'Webhook not functioning', 'event-tickets' );
+		}
+
+		// Check configuration status
+		if ( ! empty( $config_status ) ) {
+			// API version issues
+			if ( isset( $config_status['version_mismatch'] ) && $config_status['version_mismatch'] ) {
+				$issues[] = esc_html__( 'API version mismatch', 'event-tickets' );
+			}
+
+			// Event type issues
+			if ( isset( $config_status['event_types_check'] ) && isset( $config_status['event_types_check']['is_current'] ) && ! $config_status['event_types_check']['is_current'] ) {
+				$issues[] = esc_html__( 'Event types configuration outdated', 'event-tickets' );
+			}
+
+			// Missing events
+			if ( ! empty( $config_status['missing_events'] ) ) {
+				$issues[] = esc_html__( 'Missing webhook events', 'event-tickets' );
+			}
+		}
+
+		$issues_text = implode( ', ', $issues );
+
+		// Create a URL to the ticket settings page
+		$settings_url = admin_url( 'admin.php?page=tec-tickets-settings&tab=payments&section=square' );
+
+		// Create the webhook nonce
+		$webhook_nonce = wp_create_nonce( 'square-webhook-register' );
+
+		$message = sprintf(
+			'<p><strong>%1$s</strong></p><p>%2$s</p><p><a href="%3$s" class="button button-primary" data-nonce="%5$s">%4$s</a></p><div class="tec-tickets__admin-settings-square-webhook-nonce" data-nonce="%5$s" style="display: none;"></div>',
+			esc_html__( 'Square Webhook Issue', 'event-tickets' ),
+			sprintf(
+				/* translators: %s: List of webhook issues */
+				esc_html__( 'The Square payment gateway has webhook issues: %s. This may affect payment updates and order processing.', 'event-tickets' ),
+				$issues_text
+			),
+			esc_url( $settings_url ),
+			esc_html__( 'Fix Webhook Configuration', 'event-tickets' ),
+			esc_attr( $webhook_nonce )
+		);
+
+		return $message;
```

Comment:
That's a useless variable right ?

```suggestion
		return sprintf(
			'<p><strong>%1$s</strong></p><p>%2$s</p><p><a href="%3$s" class="button button-primary" data-nonce="%5$s">%4$s</a></p><div class="tec-tickets__admin-settings-square-webhook-nonce" data-nonce="%5$s" style="display: none;"></div>',
			esc_html__( 'Square Webhook Issue', 'event-tickets' ),
			sprintf(
				/* translators: %s: List of webhook issues */
				esc_html__( 'The Square payment gateway has webhook issues: %s. This may affect payment updates and order processing.', 'event-tickets' ),
				$issues_text
			),
			esc_url( $settings_url ),
			esc_html__( 'Fix Webhook Configuration', 'event-tickets' ),
			esc_attr( $webhook_nonce )
		);
```